src/commands/codepush/lib/legacy-codepush-service-client.ts

```diff
@@ -39,8 +39,12 @@ export default class LegacyCodePushServiceClient {
     return this._accessKey;
   }   
 
+  private get appIdentifer(): string {
```

Comment:
Do we really need this? `_app` instance of `DefaultApp` class already has `identifier` property, for example
https://github.com/Microsoft/appcenter-cli/blob/552e0f7d17316aaa1fc3e7fed8908c7c6d4d65b8/src/commands/apps/delete.ts#L15

Reply:
Good catch, missed this, thanks for pointing!
@buptkang agreed with what you said above! Hope this PR and other CodePush related PR's will be merged ASAP.

Reply:
Yeah, I'd prefer if you removed this and just used this._app.identifier instead. Also, since I can't make a comments on the individual lines for some reason if they're not part of the changes, some additional comments before merging this:

Line 2 - "remove `/index` from the import path, it's automatic. In general, import the directory as a package and leave it up to the package author to expose things.

Line 26: You could use parameter properties on your constructor and save a bunch of variables - for example do this instead:
```
  constructor(private accessKey: string, private serverUrl: string, private app: DefaultApp, private debug: Function, private customHeaders?: Headers)
```

And then remove all the `this._accessKey = accessKey` lines, and all the `_` properties.

Line 26: DO NOT PASS IN THE DEBUG FUNCTION! The whole point of debug is we know what line in which file it is beign printed from, this breaks that. Create your own debug function by requiring the debug module the same way every other file does it.